dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -182,7 +185,10 @@ parsePrefixMap =
       e <- Dhall.Parser.expr
       imp <- parseImport prefix e
       return (pack prefix, imp)
-    result = parse ((Dhall.Parser.unParser parser `sepBy1` char ',') <* eof) "MAPPING"
+
+    parser' = runReaderT (Dhall.Parser.unParser parser) Dhall.Util.CommentIsWhitespace
```

Comment:
Should this use `runParser`?

Reply:
Yes! I must have thought applying `unParser` before adding other combinators (`sepBy1...`) originally mattered which is why I applied the reader manually here.

But it looks like the other parser combinators apply perfectly well to our custom parser :+1: